lib/cell/view_model.rb

```diff
@@ -28,9 +28,9 @@ def self.controller_path
 
     module Helpers
       # Constantizes name if needed, call builders and returns instance.
-      def cell(name, *args, **kws, &block) # classic Rails fuzzy API.
+      def cell(name, *args, &block) # classic Rails fuzzy API.
         constant = name.is_a?(Class) ? name : class_from_cell_name(name)
-        constant.(*args, **kws, &block)
+        constant.(*args, &block)
```

Comment:
What warning does removing `**kws` resolve for you?

Reply:
This is the warning we got:
```
/home/richard/.rvm/gems/ruby-2.7.5/bundler/gems/cells-88796e3a29cb/lib/cell/view_model.rb:67: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/home/richard/.rvm/gems/ruby-2.7.5/bundler/gems/cells-88796e3a29cb/lib/cell/view_model.rb:31: warning: The called method `cell' is defined here
```
I was wondering why it wasn't failing with ruby 3 and figured out that those keyword arguments are not used because the instance method cell helper was passing the option as a positional argument anyways (so the kws hash would always be empty in ruby 3).